### ISSUE TITLE
Button element is empty and has no accessible name.

### ISSUE THREAD

**Issue by Horlix** [canonical/ubuntu.com#11890]
List of affected URLs: https://docs.google.com/spreadsheets/d/12PCiksNMTyIrb2jEbXkCrj4thtPg5eng5rYPexCAObE/edit?usp=sharing

A programmatically determined name allows screen readers to tell users what the control does. To add a name do one of the following:

Add text between the button start and end tags
Add a title attribute
Add an aria-label attribute
Add an aria-labelledby attribute
Add an img alt attribute if the button contains an img element

Example URLs:
- https://ubuntu.com/ceph/docs
- https://ubuntu.com/ceph/docs/adding-mons
- https://ubuntu.com/ceph/docs/adding-osds

Icon font needs a label.

`<button type="submit" alt="search" class="p-search-box__button"><i class="p-icon--search"></i></button>`

**Comment by Horlix** [comment]
Full set of URLs

[This button element is empty and has no accessible name.csv](https://github.com/canonical-web-and-design/ubuntu.com/files/9213927/This.button.element.is.empty.and.has.no.accessible.name.csv)
